.gitattributes

```diff
@@ -1,2 +1 @@
-* text=auto
-README.md merge=union
+* text=auto eol=lf merge=union
```

Comment:
There isn't any complicated programming language code here, so I think `merge=union` should be a safe strategy for all files.

`eol=lf` workarounds https://github.com/thlorenz/doctoc/issues/161 in case anyone uses Windows.